src/analog.jl

```diff
@@ -52,8 +49,8 @@ function analog_output(t::AOTask, channel::String; range=nothing)
         range=float(analog_output_ranges(device)[end,:])
     end
     catch_error( CreateAOVoltageChan(t.th,
-            Ref(codeunits(channel),1),
-            Ref(codeunits(""),1),
+            pointer(channel),
+            pointer(""),
```

Comment:
change this back to Ref too?

Reply:
I thought I did... On my fork it shows that its back to Ref. Do I have to do anything special to re-sync?

Reply:
`git push --force ...` should do it.  or perhaps you forgot this one.  there are three places total that your originally made the change.

Reply:
i just checked your [fork](https://github.com/bomber8013/NIDAQ.jl/blob/master/src/analog.jl#L53).  it still has `pointer`.

also, could you please squash your last four commits so as not to clutter the git log?  nominally you have made four changes here, which ideally would correspond to four commits:  1. fix the slow get_props, 2. use an enum, 3. change the order of the input args, and 4. add a default precision.  combining the last three into one is reasonable though.

Reply:
and thanks for this by the way!  it's great to have some help.

Reply:
OK, I missed that last one apparently. I don't know how to squash commits currently. 

I am happy to help! I was just told by my manager that I shouldn't be letting my company's efforts just go for free into the world so you may no get anymore of these from me, despite the development I plan on doing, unfortunately.